CONTRIBUTING.md

```diff
@@ -0,0 +1,142 @@
+# Contributing
+
+## How do I... <a name="toc"></a>
+
+- [Use This Guide](#introduction)?
+- Ask or Say Something? 🤔🐛😱
+  - [Request Support](#request-support)
+  - [Report an Error or Bug](#report-an-error-or-bug)
+  - [Request a Feature](#request-a-feature)
+- Make Something? 🤓👩🏽‍💻📜🍳
+  - [Project Setup](#project-setup)
+  - [Contributor License Agreement (CLA)](#contributor-license-agreement-cla)
+  - [Contribute Documentation](#contribute-documentation)
+  - [Contribute Code](#contribute-code)
+
+## Introduction
+
+Thank you so much for your interest in contributing!. All types of contributions are encouraged and valued. See the [table of contents](#toc) for different ways to help and details about how this project handles them!📝
+
+Please make sure to read the relevant section before making your contribution! It will make it a lot easier for us maintainers to make the most of it and smooth out the experience for all involved. 💚
+
+The Project Team looks forward to your contributions. 🙌🏾✨
+
+## Request Support
```

Comment:
Is this section valid for examples?

Reply:
If something isn't working in an example how else could someone ask for assistance?

Reply:
Bug report is below - for this report in particular I can't see what the difference would be?